docs/install.md

```diff
@@ -21,12 +21,6 @@ stream.
   procedures for a newer distribution. The newly installed version will become
   the default.
 
-    In SAT 2.4 and newer, you can instead upgrade the product stream by using the
```

Comment:
This is a really small nitpick, but now in the release/2.4 branch of the docs it will say "In SAT 2.4 and newer..." which is no longer true, it would just be "in SAT 2.4..."

I am really not concerned about patching the old release branches though, especially for such a small thing.

Reply:
I agree it's not super important, but I think we should be accurate. I created [STP-3519](https://jira-pro.its.hpecorp.net:8443/browse/STP-3519) to work on this later.